src/main/java/io/camunda/getstarted/whosfifty/Worker.java

```diff
@@ -37,6 +37,7 @@ public void CheckCelebAge(final JobClient client, final ActivatedJob job) {
                 HashMap<String, Object> variables = new HashMap<>();
                 variables.put("age", celebAge);
                 variables.put("celebName", celebName);
+                variables.put("birthday", celebDetails.birthday);
```

Comment:
It's useful to show the actual birthday, I think every time someone wins, we want to double check it to see if it's actually true or not :D